.github/workflows/devbuilds.yml

```
@@ -0,0 +1,44 @@
+# Create Development Builds
+name: Development Builds
+on:
+  # Runs on push in repository or pull requests
+  push:
+  pull_request:
+
+jobs:
+  prepare:
+    name: Prepare Build
+    runs-on: ubuntu-lastest
```

Is it being REALLY dumb and unhelpful and waiting for a runner type that will never exist?
```suggestion
    runs-on: ubuntu-latest
```

---

Well, that's a stupid mistake...

But it works now.